tests/lib/rules/object-shorthand.js

```diff
@@ -516,6 +516,16 @@ ruleTester.run("object-shorthand", rule, {
             output: null,
             errors: [METHOD_ERROR]
         },
+        {
+            code: "var x = {a: /* comment */ a}",
+            output: null,
+            errors: [PROPERTY_ERROR]
+        },
+        {
+            code: "var x = {a /* comment */: a}",
+            output: null,
+            errors: [PROPERTY_ERROR]
+        },
```

Comment:
Can you also add tests with quotes to make sure that case is covered.

```suggestion
        },
        {
            code: "var x = {'a': /* comment */ a}",
            output: null,
            errors: [PROPERTY_ERROR]
        },
        {
            code: "var x = {'a' /* comment */: a}",
            output: null,
            errors: [PROPERTY_ERROR]
        },
```

Reply:
[6005b24](https://github.com/eslint/eslint/pull/18442/commits/6005b24af15e1dd5e96a7482a866ee1bee3b5170)